docs/compendium/substitute.md

```diff
@@ -70,12 +70,81 @@ input of request-body for substitution
 {"error":"overlay ERC not found"}
 ```
 
+## Run substitution
+
+`Run substitution` will produce a new ERC with metadata, saved to mongoDB, and also run the substitution in a docker container.
+
+### Request
+
+`POST /api/v1/substitution/run`
```

Comment:
No need for `/run` in the end. `POST` already carries the meaning of "create a new one".